yksgi/src/lib.rs

```diff
@@ -156,11 +156,18 @@ impl SGInterp {
         instridx: usize,
         fname: &CStr,
         sfidx: usize,
-        val: u64,
+        mut val: u64,
     ) {
         let func = self.module.function(fname.as_ptr());
         let bb = func.bb(bbidx);
         let instr = bb.instruction(instridx);
+
+        if instr.get_type().is_integer() {
+            // Discard undefined high-order bits in stackmap constants.
```

Comment:
Should we add some more detail here so we don't forget in the future?

Reply:
Fixed in b8873c8a1d936214